inputs_supply.md

```diff
@@ -0,0 +1,167 @@
+| Capacity factor | Ratio of generation to capacity | Assumption |  | Specified by technology and year |  Unitless |
+| Fixed charge rate | Factor used to levelize capital cost | Assumption |  | Specified by technology |  Unitless |
+| Default efficiencies | Default amount of output produced per unit of input; can be overwritten by region-specific information derived from historical data | Assumption | | Specified by technology and year | GJ per GJ |
+| Default input-output coefficients | Default amount of input required per unit of output produced; can be overwritten by region-specific information derived from historical data | Assumption | | Specified by technology and year |  GJ per GJ |
+| Resource supply curves | Mapping between cost and resource extraction. Resource extraction is cumulative for deplatable resources and annual for renewable resources | External data | Various | Specified by resource and year |  EJ for extraction, 1975$/GJ for cost |
+
+
+Table 1: External inputs used for supply of energy<sup>[1](#table_footnote1)</sup>
+
+Note that for the Shared Socioeconomic Pathways (SSPs), different inputs are used for some variables. See [SSPs](ssp.html) for more information.
+
+#### Data
+
+Throughout GCAM, the number in the name of assumption file indicates to which sector the file applies. Files with `A10` through `A20` in the name are assumptions for resources. Files with `A21` and `A22` in the name are assumptions for refining and gas processing. Files with `A23` in the name are assumptions for electricity, `A24` is for heat, and `A25` is for hydrogen. Files with `A26` in the name are assumptions for transmission and distribution.
+
+##### Historical supply of energy
+
+GCAM uses IEA energy balances as a source for historical energy supply and demand. IEA data are proprietary and thus are not provided in the GCAM data repository. Instead, we provide all of the `R` code used to process the IEA data so that the user can replicate the processing _if_ they purchase the IEA data. In addition, we provide aggregated data after it has undergone processing so that GCAM input files can be created and used by the user community. 
+
+##### CO<sub>2</sub> capture rates
+
+CO<sub>2</sub> capture rates for cement are specified in [A25.globaltech_co2capture.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A25.globaltech_co2capture.csv). Capture rates for fertilizer are specified in [A23.globaltech_co2capture.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A23.globaltech_co2capture.csv)
+
+##### Retirement rules
+
+Retirement rules are specified in [A22.globaltech_retirement.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A22.globaltech_retirement.csv), [A23.globaltech_retirement.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A23.globaltech_retirement.csv)
+
+
+##### Logit exponents
```

Comment:
If we're including carbon storage in energy supply, this list should include A61.sector.csv, and [A61.subsector_logit.csv] (https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A61.subsector_logit.csv).

Also, if we're including fossil fuel trade sectors here, we need A_ff_RegionalSector.csv, A_ff_RegionalSubsector.csv, and A_ff_TradedSector.csv.